### PR TITLE
rlvm 0.14 (new formula)

### PR DESCRIPTION
**Pull request by tomyun**
[rlvm](http://www.rlvm.net/) is an open-source implementation of RealLive scripting engine.

- `inreplace` to use brewed shared libraries

### CODE REVIEW

File: rlvm.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      s.gsub! /(static_env\.ParseConfig)\("sdl-config --static-libs", MergeEverythingButSDLMain\)/, '\1("pkg-config --libs sdl SDL_image SDL_mixer SDL_ttf freetype2").Append(FRAMEWORKS=["OpenGL"])'
+      s.gsub! /(full_static_build) = True/, '\1 = False'
+    end
+    scons "--release"
+    prefix.install "build/rlvm.app"
+    bin.write_exec_script "#{prefix}/rlvm.app/Contents/MacOS/rlvm"
+  end
+
+  test do
+    File.executable? "#{prefix}/rlvm.app/Contents/MacOS/rlvm"
```
Comment (bfontaine) [inline rlvm.rb]: Please write a test for the actual functionality.

**Comment by mistydemeo** [comment]
Ping?

File: rlvm.rb (hunk 1 above)
Comment (tomyun) [inline rlvm.rb]: `rlvm` seems to accept no arguments that can be used for testing. I'll remove `test` block.

**Comment by mistydemeo** [comment]
Thank you!

Minor nitpick, but we prefer dependencies to be alphabetized within each section. I updated the commit while merging it.

**Pull request closed without merging**